students/yooyoung/users/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.core.validators import RegexValidator
+
+
+class User(models.Model):
+    name = models.CharField(max_length=20)
+    email = models.EmailField(max_length=50, unique=True)
+    password = models.CharField(max_length=50)
+    note = models.TextField(max_length=100, null=True)
+    phone = models.CharField(
+        max_length=16,
+        blank=True,
+        null=True,
+        validators=[
+            RegexValidator(
+                regex=r'^\+?1?\d{9,15}$',
+                message="Phone number must be entered in the format '+123456789'. Up to 15 digits allowed."
+            ),
+        ],
+    )
+    
+    class Meta:
+        db_table = 'Users'
```

Comment:
위코드 컨벤션으로 database 테이블명은 클래스명의 소문자, 복수형을 따르고 있습니다!
```suggestion
        db_table = 'users'
```